paddle/go/pserver/service.go

```diff
@@ -42,15 +43,16 @@ type Gradient Parameter
 type Service struct {
 	initialized chan struct{}
 
-	mu       sync.Mutex
-	opt      *optimizer
-	paramMap map[string]Parameter
+	mu           sync.Mutex
+	paramMap     map[string]Parameter
```

Comment:
Since there is already optimizerMap, and optimizer owns parameter. So maybe we no longer need paramMap?